frontend/src/component/changeRequest/DraftBanner/DraftBanner.tsx

```diff
@@ -65,7 +65,7 @@ const DraftBannerContent: FC<{
 const StickyBanner = styled(Box)(({ theme }) => ({
     position: 'sticky',
     top: -1,
-    zIndex: theme.zIndex.appBar,
+    zIndex: 200 /* has to lower than header.zIndex */,
```

Comment:
It would be nice to keep it linked to `theme`

https://mui.com/material-ui/customization/default-theme/#main-content

Reply:
unfortunately the header has index of 300 (literal value, not from a theme) and I don't know what the impact may be of touching a header zIndex.